MIGRATING.md

```diff
@@ -0,0 +1,60 @@
+# Migration guides
+
+## 0.5 -> 0.6 migration
+
+The 0.6 migration includes two breaking changes and some useful new functionality:
+
+1\. The `Result.Ok()` and `Result.Err()` class methods have been removed. 
+These should be replaced by direct use of `Ok()` and `Err()`. As an example, the following code:
+
+```python
+from result import Result
+res1 = Result.Ok('yay')
+res2 = Result.Err('nay')
+```
+
+should be replaced by:
+
+```python
+from result import Ok, Err
+res1 = Ok('yay')
+res2 = Err('nay')
+```
+
+2\. Result is now a Union type between `Ok[T]` and `Err[E]`. As such, you cannot use `isinstance(res, Result)` anymore.
+These should be replaced by `isinstance(res, Result_)`. As an example, the following code:
```

Comment:
There's still a `Result_` here that you forgot :slightly_smiling_face:

Reply:
Fixed in 7419c05bd1697c406f9aa22481622dcbbd283294